src/client/visualizations/heat-map/heatmap-hover-tooltip.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2017-2018 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TooltipWithBounds } from "@vx/tooltip";
+import { Datum } from "plywood";
+import * as React from "react";
+import { Essence } from "../../../common/models/essence/essence";
+import { ConcreteSeries, SeriesDerivation } from "../../../common/models/series/concrete-series";
+import { MeasureBubbleContent } from "../../components/measure-bubble-content/measure-bubble-content";
+import { SegmentBubbleContent } from "../../components/segment-bubble/segment-bubble";
+import datumByPosition from "./utils/datum-by-position";
+import { HoverPosition } from "./utils/get-hover-position";
+import { modalTitle } from "./utils/modal-title";
+
+interface HeatmapHoverTooltip {
+  dataset: Datum[];
+  position: HoverPosition;
+  essence: Essence;
+  scroll: { left: number, top: number };
+}
+
+interface ContentProps {
+  datum: Datum;
+  showComparison: boolean;
+  series: ConcreteSeries;
+}
+
+const Content: React.SFC<ContentProps> = props => {
+  const { showComparison, series, datum } = props;
+  if (!showComparison) {
+    return <React.Fragment>{series.formatValue(datum)}</React.Fragment>;
```

Comment:
Why do you need a fragment here? Types?

Reply:
Hmm, dunno.

Reply:
Yup, React typings require that we have `ReactElement` here and string is not part of this type.